client/src/lib.rs

```diff
@@ -15,12 +15,14 @@ pub mod samples {
     use super::Configuration;
 
     /// Get sample client configuration.
+    #[allow(clippy::expect_used)]
     pub fn get_client_config(key_pair: &KeyPair) -> Configuration {
         let (public_key, private_key) = key_pair.clone().into();
         Configuration {
             public_key,
             private_key,
-            account_id: iroha_data_model::prelude::AccountId::test("alice", "wonderland"),
+            account_id: iroha_data_model::prelude::AccountId::new("alice", "wonderland")
+                .expect("Should not fail."),
```

Comment:
why this change? you consider it more safe?

Reply:
also, this method is only used in tests and we should find a way to exclude it from release builds

Reply:
Here's my opinion. Ideally, we shouldn't ever be skipping any checks in either tests or benchmarks. They either have negligible overhead, and having checked and unchecked variants is unnecessary complexity, or they have significant overhead, which we would see in the real world too.

Reply:
yeah, I think my comment was offtopic. We can close this. However, it should be fine to use `unwrap` in tests. Also once `Option::expect` or/and `Option::unwrap` are const functions I think this should remove the necessity to use `static Lazy` which I've seen around.

Reply:
I want to disable unwrap globally. 

We only have expect for cases when an invariant is enforced and so something either never panics, or if it does, points to the violated invariant. Unwrap gives you no such information, and I think we can live without it. 

In tests the panic message should point you to the invariant, even if the invariant is "this string is a valid name", hence `.expect("valid")`.